conf/application-json-logger.xml

```diff
@@ -6,8 +6,4 @@
     </appender>
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
-
-    <root level="${logger.application:-WARN}">
-        <appender-ref ref="STDOUT"/>
-    </root>
```

Comment:
Won't this prevent us from seeing application logs now?

Reply:
Reverted